ext/bin/load-test/config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: v4/facts GET
+  path: "/pdb/query/v4/facts"
+  query: query=["~", "certname", "host*"]
+  method: GET
+  headers: ''
+- name: post call
+  path: "/pdb/query/v4/facts"
+  query: {"query": ["~", "certname", "host"]}
+  method: POST
+  headers: {content-type: application/json}
+- name: v4/facts GET2
+  path: "/pdb/query/v4"
+  query: query=["from", "resources", ["extract", ["certname"], ["and", ["and", ["=", "type", "Class"], ["or", ["=", "title", "Puppet_enterprise::Profile::Certificate_authority"], ["or", ["=", "title", "Puppet_enterprise::Profile::Master"], ["=", "title", "Puppet_enterprise::Profile::Console"]], ["=", "title", "Puppet_enterprise::Profile::Puppetdb"]], ["subquery", "nodes", ["and", ["null?", "deactivated", true], ["null?", "expired", true]]]], ["=", "node_state", "active"]]]]
+  method: put
```

Comment:
This should be a POST method

Reply:
This was meant to be an example. Updated.